internal/engine/upper.go

```diff
@@ -12,10 +12,11 @@ import (
 
 	"github.com/opentracing/opentracing-go"
 	"github.com/pkg/errors"
-	"github.com/windmilleng/tilt/internal/sail/client"
 	v1 "k8s.io/api/core/v1"
 	"k8s.io/apimachinery/pkg/labels"
 
+	"github.com/windmilleng/tilt/internal/sail/client"
+
```

Comment:
Delete this empty line?

Reply:
That annoys me all the time. I wonder if goimports does that intentionally or if it'd be possible to fix that...